builder/amazon/chroot/run_local_commands.go

```diff
@@ -0,0 +1,37 @@
+package chroot
+
+import (
+	"fmt"
+
+	"github.com/mitchellh/packer/packer"
+	"github.com/mitchellh/packer/post-processor/shell-local"
```

Comment:
Not sure how people feel about this dependency, but I felt it was better than copying the communicator from the shell-local post-processor.  I think the best option would be to extract all of heavily duplicated shell logic somewhere else where it can be re-used between provisioners, post-processors and builders, but I didn't want to tackle that in this PR.